pkg/sdkserver/sdkserver.go

```diff
@@ -1230,8 +1230,8 @@ func (s *SDKServer) updateList(ctx context.Context) error {
 
 	// Cache a copy of the successfully updated gameserver
 	s.gsCopy = gs
-	// Clear the gsCounterUpdates
-	s.gsCounterUpdates = map[string]counterUpdateRequest{}
+	// Clear the gsListUpdates
+	s.gsListUpdates = map[string]listUpdateRequest{}
```

Comment:
Do we need an update to a unit test to ensure this doesn't happen again?

Reply:
I'll have a look in the unit test file to see what's doable.

Reply:
Appreciate you taking a look.

No rush, but just a heads up - we're cutting a release next Tuesday, so if you want this in the next release, we should get some unit tests in.

@igooch any advice on a good spot to write a unit test?

Reply:
I have something actually but it's at work, so tomorrow I should send an update to this MR, idea is:

- add a function that get the len() of the outgoing lists
- happy path = len() == 0
- network failure / issues updating = original len

Reply:
Looks good! The only thing I'd add is checking that the `gsListUpdates` has the update, i.e. is not zero after the update/add/remove call and before the get list call. Something like

```
if testCase.updated {
  updates := sc.gsListUpdatesLen()
  assert.Equal(t, 1, updates)
}
```

Reply:
^Scratch that above suggestion, it would introduce a race condition. Code looks good as it stands.